go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -695,7 +694,7 @@ func getMgmtPortAndRepName(node *corev1.Node) (string, string, error) {
 	}
 }
 
-func createNodeManagementPorts(node *corev1.Node, nodeLister listers.NodeLister, nodeAnnotator kube.Annotator, kubeInterface kube.Interface, waiter *startupWaiter,
```

Comment:
I am a bit confused.... why didn't the "unused parameter" lint commit catch this? I understand this commit is coming from https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5077/commits/882deacba6fd821046154c2e7477226b5cfd798f#r1960611793 this review comment but does that mean the linter is not smart enough to catch those?